.gitignore

```diff
@@ -2,5 +2,4 @@ build/
 install/
 log/
 config/
-.vscode/
-imgui.ini/
```

Comment:
I think this is the users customization of where and how big the boxes are, which is why I added it to .gitignore

Reply:
I wasn't able to run the GUI without it...